common/src/main/proto/pb.proto

```diff
@@ -1378,6 +1378,7 @@ message PreferencesPayload {
     int32 ignore_dust_threshold = 51;
     double buyer_security_deposit_as_percent_for_crypto = 52;
     int32 block_notify_port = 53;
+    bool use_dark_mode = 54;
```

Comment:
I think it would be better to not use a boolean in case we'll add more color variations in the future (see https://github.com/bisq-network/bisq/pull/2414/files#diff-c8de125bfd3c56b7fa8f2bca445dd516)

Reply:
Okay, will change to int